sections/futures/Trade/TradeConfirmationModal.tsx

```diff
@@ -39,13 +46,25 @@ export default function TradeConfirmationModal({
 
 	const market = useRecoilValue(currentMarketState);
 	const { data: potentialTradeDetails } = useRecoilValue(potentialTradeDetailsState);
+	const orderType = useRecoilValue(orderTypeState);
+	const orderPrice = useRecoilValue(futuresOrderPriceState);
+	const position = useRecoilValue(positionState);
+
+	const positionSide = useMemo(() => {
+		if (potentialTradeDetails?.size.eq(zeroBN)) {
+			return position?.position?.side === PositionSide.LONG
+				? PositionSide.SHORT
+				: PositionSide.LONG;
+		}
+		return potentialTradeDetails?.size.gte(zeroBN) ? PositionSide.LONG : PositionSide.SHORT;
+	}, [potentialTradeDetails, position?.position?.side]);
 
 	const positionDetails = useMemo(() => {
```

Comment:
The `protocolFee` value does not include the stop/limit order fee. The modal displays protocol fee which only includes the trading fee and cross margin fee.